images/main/Dockerfile

```diff
@@ -12,6 +12,6 @@ WORKDIR /app
 COPY ./src /app
 COPY ./scripts /app/scripts
 
-RUN pip install -r requirements.txt
+RUN pip install -r requirements.pgonly.txt
```

Comment:
My 2 cents: call it `requirements.postgres.txt` and `requirements.mysql.txt`. So it's easier to understand. But not a blocker.